sox.features.js

```diff
@@ -35,45 +35,57 @@
     },
 
     markEmployees: function () {
-      // Description: Adds an Stack Overflow logo next to users that *ARE* a Stack Overflow Employee
+      // Description: Adds the Stack Exchange logo next to users that *ARE* Stack Exchange employees
 
-      const anchors = [...document.querySelectorAll('.comment a, .deleted-answer-info a, .employee-name a, .user-details a, .question-summary .started a')].filter(el => {
-        return !el.parentElement.classList.contains('user-gravatar32') && el.href && el.href.contains('/users/');
-      });
-      const ids = [];
+      const $icon = $('<svg aria-hidden="true" class="sox-markEmployees-logo sox-sprite svg-icon iconStackExchange" width="18" height="18" viewBox="0 0 18 18"><path d="M15 1H3a2 2 0 0 0-2 2v2h16V3a2 2 0 0 0-2-2zM1 13c0 1.1.9 2 2 2h8v3l3-3h1a2 2 0 0 0 2-2v-2H1v2zm16-7H1v4h16V6z"></path></svg>');
+      $icon.css('color', $('.mod-flair').css('color'));
 
-      for (let i = 0; i < anchors.length; i++) {
-        const href = anchors[i].href;
-        const idMatch = href.match(/(\d+)/);
-        if (idMatch && ids.indexOf(idMatch[1]) === -1) ids.push(idMatch[1]);
-      }
-      sox.debug('markEmployees user IDs', ids);
+      function getIds() {
+        const anchors = [...document.querySelectorAll('.comment a, .deleted-answer-info a, .employee-name a, .user-details a, .question-summary .started a')].filter(el => {
+          return !el.parentElement.classList.contains('user-gravatar32') && $(el).find('.sox-markEmployees-logo') && el.href && el.href.contains('/users/');
```

Comment:
Should this addition be `!$(el).find('.sox-markEmployees-logo').length` instead, so we exclude anchors with the logo already added?